packages/shared/src/graphql/squads.ts

```diff
@@ -14,6 +17,16 @@ export type Squads = {
   squads: Squad[];
 };
 
+export enum SquadMemberRole {
+  Member = 'member',
+  Role = 'role',
```

Comment:
Were there any specifics with regard to the roles available on the initial rollout? I think we will have come up with a more accurate name.

Reply:
We have owner and member at the moment.
https://github.com/dailydotdev/daily-api/blob/5c7f80ec191e6cc267c53a3fd858bef8c11719e2/src/entity/SourceMember.ts#L9

Reply:
This should have been Owner instead of Role. My mistake. Fixed